src/Compilers/Core/MSBuildTask/ManagedCompiler.cs

```diff
@@ -535,7 +547,10 @@ protected internal virtual void AddResponseFileCommands(CommandLineBuilderExtens
             commandLine.AppendPlusOrMinusSwitch("/optimize", this._store, "Optimize");
             commandLine.AppendSwitchIfNotNull("/out:", this.OutputAssembly);
             commandLine.AppendSwitchIfNotNull("/ruleset:", this.CodeAnalysisRuleSet);
+            commandLine.AppendSwitchIfNotNull("/errorlog:", this.ErrorLog);
             commandLine.AppendSwitchIfNotNull("/subsystemversion:", this.SubsystemVersion);
+            // TODO: uncomment the below line once "/reportanalyzer" switch is added to compiler.
```

Comment:
@heejaechang you will need to uncomment this once this switch is added to compiler.